app/components/primer/alpha/text_field.pcss

```
@@ -207,7 +207,7 @@
   }
 }
 
-.FormControl-toggleSwitchInput {
+.FormControl-toggleSwitchInput, toggle-switch-input.FormControl-toggleSwitchInput {
```

One other option here is this neat css trick that will strengthen the specificity.

```suggestion
.FormControl-toggleSwitchInput.FormControl-toggleSwitchInput {
```